install.sh

```diff
@@ -26,17 +29,19 @@ version_ge() {
 	[[ "${1%.*}" -gt "${2%.*}" ]] || [[ "${1%.*}" -eq "${2%.*}" && "${1#*.}" -ge "${2#*.}" ]]
 }
 
-OS="$(uname)"
-if [[ "${OS}" != "Linux" ]] && [[ "${OS}" != "Darwin" ]]; then
-	abort "Sail CLI is only supported on macOS and Linux. Windows users can use Sail via WSL."
-fi
-
 test_python() {
 	local python_version_output
 	python_version_output="$("$1" --version 2>/dev/null)"
 	version_ge "$(major_minor "${python_version_output##* }")" "$(major_minor "${REQUIRED_PYTHON_VERSION}")"
 }
 
+/usr/bin/sudo -l mkdir &>/dev/null || abort "This installer requires root or sudo."
```

Comment:
Again, assumption of sudo location, as well as its existence. If the script is run as root and sudo is not available we should still be ok.

Reply:
Should I wrap this in `if [ ! "$UID" -eq 0 ]; then` too?

Reply:
I guess so, unless you can think of a better way to do it.

Reply:
Done 🍏